source/clear-linux/tutorials/telemetry-e2e.rst

```diff
@@ -0,0 +1,294 @@
+.. _telemetry-e2e:
+
+
+|CL-ATTR| includes a telemetry and analytics solution (also known as telemetrics) as part of the OS, which records events of interest and reports them back to the development team using the telemetrics client daemons.
+End users can enable or disable the telemetry client component of |CL| and also redirect where records are sent if they wish to collect records for themselves by using their own telemetry backend server.
+
+This tutorial walks you through creating a telemetry enabled app on your local |CL| machine. The tutorial will show you how to send your own custom reports to any telemetry backend server.
```

Comment:
I would continue to use records instead of reports

Reply:
Agree! Good catch.